docker-compose.yml

```diff
@@ -145,6 +145,8 @@ services:
       IDOX_NEXUS_TOKEN_URL: ${IDOX_NEXUS_TOKEN_URL}
       JWT_SECRET: ${JWT_SECRET}
       MAPBOX_ACCESS_TOKEN: ${MAPBOX_ACCESS_TOKEN}
+      METABASE_API_KEY: ${METABASE_API_KEY}
+      METABASE_URL_EXT: ${METABASE_URL_EXT}
```

Comment:
We also need to pass this along to the Fargate API instance here - https://github.com/theopensystemslab/planx-new/blob/main/infrastructure/application/index.ts#L331-L456

Reply:
Pushed a new commit with this, hope I have done / understood correctly?